qml/plugins/CleanTracker.qml

```diff
@@ -7,12 +7,15 @@ ActionItem {
     title: qsTrId("sailfish-tools-he-tracker_database")
     //% "Clean"
     actionName: qsTrId("sailfish-tools-bt-clean")
+    //: Text surrounded by %1 and %2 is a link pointing to documentation
     //% "Tracker dabatabase cleaning can help in cases with "
     //% "missing images, audio files etc. Processes using tracker "
     //% "will be closed, tracker reindexing will be started. "
     //% "More information can be found "
-    //% "<a href='https://together.jolla.com/question/4337/refresh-tracker/'>here</a>."
+    //% "%1here%2."
     description: qsTrId("sailfish-utilities-me-clean-tracker-db-desc")
+        .arg("<a href='docs.sailfishos.org/Support/Help_Articles/Tips_and_Tricks/#refresh-the-tracker-database'>")
```

Comment:
Just the `https://` missing from in front of the URL I think (it's needed, it seems, to make the link work).

Reply:
:facepalm: Thanks. I'll change the extra comment text as well to follow sledges' suggestion